encoding/ssz/query/analyzer.go

```diff
@@ -30,6 +31,80 @@ func AnalyzeObject(obj any) (*sszInfo, error) {
 	return info, nil
 }
 
+// PopulateFromValue receives an actual value and fills the sszInfo dynamically.
+func PopulateFromValue(sszInfo *sszInfo, value any) error {
```

Comment:
I think the function's purpose is not documented enough, and the name and/or description can be improved. This function populates information concerning variable-length items - can you explicitly mention this?